build-config/scripts/make-devices.pl

```diff
@@ -168,18 +168,11 @@ sub devices
     mkdir "${devdir}pts", 0755;
 
     print "NOR flash access: mtd mtdblock\n";
-    foreach my $i (0..7) {
+    foreach my $i (0..12) {
 	makedev("mtdblock${i}", "b", 31, ${i},     ${root}, ${root}, 0660);
 	makedev("mtd${i}",      "c", 90, (2*${i}), ${root}, ${root}, 0660);
     }
 
-    print "UBI device access: ubi\n";
-    makedev("ubi_ctrl", "c", 10, 63, ${root}, ${root}, 0660);
-    makedev("ubi0"    , "c", 253, 0, ${root}, ${root}, 0660);
-    foreach my $i (0..7) {
-	makedev("ubi0_${i}", "c", 253, ${i}+1, ${root}, ${root}, 0660);
-    }
-
```

Comment:
My problem with taking this out now is that for some legacy PowerPC platforms the above numbers are correct and are some OS installers are relying on it.